models/articles.models.js

```diff
@@ -14,4 +14,25 @@ exports.fetchArticleById = (article_id) => {
 
         }
     })
+}
+exports.fetchArticles = (sortBy = "created_at", order = 'desc') => {
+    const allowedOrders = ['asc', 'desc']
+    let sql = (`
+    SELECT 
+    articles.article_id,
+    articles.author,
+    articles.title,
+    articles.topic,
+    articles.created_at,
+    articles.votes,
+    articles.article_img_url,
+    COUNT(comments.comment_id) AS comment_count
+    FROM articles
+    LEFT JOIN comments ON articles.article_id = comments.article_id
+    GROUP BY articles.article_id`)
+    if(sortBy){
+        sql += ` ORDER BY ${sortBy} ${order}`
+    }
+    return db.query(sql)
+    
```

Comment:
Great job with this query 👍 

If you're interested, there is a way of casting a column to be a different data type. I can see comment_count is a string, you can add a way of casting it to be a number. Not necessary and you could just change this in the front end in future - but might be nice.